### DISCUSSION TITLE
这个pcl2服务器重要吗，看每次都这样

### DISCUSSION THREAD

**Discussion by Chormore-EDGE** [Meloong-Git/PCL#1385] [提问] [answered]
![屏幕截图 2023-03-03 172329](https://user-images.githubusercontent.com/114737601/222713741-cf5f81a5-f200-4fba-90f8-01940649dcd6.png)

**Comment by Big-Cake-jpg** [comment, marked as answer]
这不是服务器，是 PCL2 使用的腾讯云对象存储桶。

PCL2 通过它来下载启动器更新、获取启动器公告等，如果无法访问该地址，那么 PCL2 可能无法进行自动更新。

**Comment by Chormore-EDGE** [comment]
所以我可不可以提一个issues

**Reply by Chiloven945** [reply]
龙猫好像说过不搞服务器是怕被攻击（

**Reply by Chormore-EDGE** [reply]
提了（被关了，感谢回答awa
